core/src/test/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtilTest.java

```diff
@@ -500,6 +501,11 @@ void isAbsoluteUri() {
         bad.forEach(path -> assertThat(ArmeriaHttpUtil.isAbsoluteUri(path)).isFalse());
     }
 
+    @Test
+    void serverHeader() {
+        assertThat(ArmeriaHttpUtil.SERVER_HEADER).contains("Armeria/");
```

Comment:
How about using some regular expression?

Reply:
Yeap, that sounds good.